src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/nochangeincurrentposition/NoPositionChangeCcdDataSourceTests.java

```diff
@@ -55,24 +56,23 @@ public void shouldReturnMultipleSearchResults() throws IOException {
         var authToken = "A test token";
         var caseTypeId = "A test case type";
         var ccdClient = mock(CcdClient.class);
-        var searchResult = new ArrayList<SubmitMultipleEvent>();
-        searchResult.add(new SubmitMultipleEvent());
-        when(ccdClient.buildAndGetElasticSearchRequestWithRetriesMultiples(anyString(), anyString(), anyString()))
+        var searchResult = new MultipleCaseSearchResult();
+        when(ccdClient.runElasticSearch(anyString(), anyString(), anyString(), eq(MultipleCaseSearchResult.class)))
                 .thenReturn(searchResult);
 
         var ccdReportDataSource = new NoPositionChangeCcdDataSource(authToken, ccdClient);
         var results = ccdReportDataSource.getMultiplesData(caseTypeId, new ArrayList<>());
 
-        assertEquals(1, results.size());
```

Comment:
Does it test the scenario when searchResult is null?

Reply:
added more scenarios to cover null